oauth.go

```diff
@@ -1,9 +1,46 @@
 package fortnox
 
 import (
+	"context"
+	"errors"
+
+	"github.com/go-redis/redis/v9"
 	"golang.org/x/oauth2"
 )
 
+const (
+	rdsTokenKey = "fortnox_token"
+)
+
+var (
+	ErrNoTokenInTokenStorage = errors.New("no token in token storage")
```

Comment:
Would be good to prefix all errors coming from fortnox repo with "fortnox: no token in token storage"